packages/SwingSet/test/snapshots/test-xsnap-store.js.md

```diff
@@ -12,8 +12,8 @@ Generated by [AVA](https://avajs.dev).
 
 > after SES boot - sensitive to SES-shim, XS, and supervisor
 
-    'dd40da432d48e6e6b9530c8ae3712eb8d4e6c9f98b8e96c6d3638d1c89f2b8b9'
+    'e48339a392bbcf34fe316dc20fe8bb218f6ab39b9022e462c125ab1e12ebd18d'
```

Comment:
oh right, because the supervisor imports `message.js`, which imports `capdata.js`, which was changed